infra/terraform/global/assets/prune_ebs_snapshots/README.md

```diff
@@ -0,0 +1,48 @@
+## Prune EBS Snapshots
+
+Prerequisites
+-------------
+
+You'll also need a working [Go](www.golang.org/doc/) environment to build a distribution
+
+For working with the [Go](www.golang.org) runtime in AWS Lambda. See [lambda-go-how-to-create-deployment-package](https://docs.aws.amazon.com/lambda/latest/dg/lambda-go-how-to-create-deployment-package.html)
+
+To Build and Deploy
+---------
+
+The Lambda execution environment uses a Linux kernel, so you'll need to build the binary for linux
+```
+GOOS=linux go build main.go 
```

Comment:
You may want to add the `-o prune_ebs_snapshots` so you don't need to rename/`mv` the file later.

Reply:
You're right.  Better yet! I'll just rename the file.  As long as the package is `main`. ta